stellarphot/gui_tools/comparison_functions.py

```diff
@@ -346,8 +350,27 @@ def _set_object(self):
             self._tess_object_info.layout.visibility = "hidden"
             self.tess_save_toggle.value = False
             self.tess_save_toggle.disabled = True
+            self.tess_save_toggle.layout.visibility = "hidden"
+
+        # Maybe this is a tess object?
+
+        if self.observatory is None or self.observatory.TESS_telescope_code is None:
+            _settings_for_no_tess()
+            return
+
+        try:
+            self.tess_submission = TessSubmission.from_header(
+                self._file_chooser.header,
+                telescope_code=self.observatory.TESS_telescope_code,
+                planet=1,
+            )
+        except ValueError:
+            # Guess not, time to turn on the coordinates box
+            # self._turn_on_coordinates()
+            _settings_for_no_tess()
         else:
```

Comment:
Since I always forget this, I might add a comment line after `else` to indicate this bit of code gets run if `try` succeeds.  Or not.  I constantly do reminders like this for features I use less often.